app/assets/javascripts/initializePage.js

```diff
@@ -83,8 +87,3 @@ function callInitializers() {
   // Initialize data-runtime context to the body data-attribute
   document.body.dataset.runtime = Runtime.currentContext();
 }
-
-function initializePage() {
-  initializeLocalStorageRender();
```

Comment:
There were two independent parts of this PR - one is removing this redundant call to initializeLocalStorageRender here - it's the only part I'm attached to - inlining the function, then separating it back out again, is just shuffling code and lower value.

I'm very receptive to a review that suggests dropping the second and third commits and only keeping the one line first commit.